src/profile/ProfileForm.tsx

```diff
@@ -37,585 +23,315 @@ interface ChangedUser {
+              maxLength: {
+                value: 150,
+                message: "USER.LAST_NAME.TOO_LONG.150",
+              },
+              required: true,
+            })}
           />
-          <Form.Control.Feedback type="invalid">
-            <ul>
-              {state.errors.email?.map((msg) => (
-                <li key={msg}>{ERRORS[msg]}</li>
-              ))}
-            </ul>
-          </Form.Control.Feedback>
         </Col>
       </Form.Group>
-
-      <Form.Group as={Row} controlId="profileFirstLastName">
+      <Form.Group as={Row} controlId="email">
         <Form.Label column sm={2}>
-          Name
+          Email:
         </Form.Label>
         <Col sm={10}>
-          <Form.Control
-            name="name"
-            placeholder="e.g. Johnny Appleseed"
-            value={
-              state.changedUser.name ??
-              (user
-                ? `${user.first_name}${
-                    user.last_name ? ` ${user.last_name}` : ""
-                  }`
-                : "")
-            }
-            onChange={onChange}
-            isValid={"first_name" in state.changedUser}
+          <Field
+            errors={props.context.errors}
+            type="email"
+            name="email"
+            ref={props.context.register({
+              required: "USER.REQUIRED.EMAIL",
+              maxLength: {
+                value: 254,
+                message: "USER.EMAIL.TOO_LONG.254",
+              },
+              pattern: {
+                value: /\S+@\S+\.\S+/,
+                message: "USER.EMAIL.INVALID",
+              },
+            })}
           />
         </Col>
       </Form.Group>
-
-      <Form.Group as={Row} controlId="profileWriterName">
+      <Form.Group as={Row} controlId="writerName">
         <Form.Label column sm={2}>
-          Writer name
+          Writer Name:
         </Form.Label>
         <Col sm={10}>
-          <Form.Control
+          <Field
+            errors={props.context.errors}
+            type="text"
             name="writer_name"
-            placeholder="e.g. Grower of Apples"
-            value={state.changedUser.writer_name ?? user?.writer_name ?? ""}
-            onChange={onChange}
-            isValid={(state.changedUser.writer_name?.length ?? 0) > 0}
-            isInvalid={
-              state.errors.writer_name && state.errors.writer_name.length > 0
-            }
+            ref={props.context.register}
           />
-          <Form.Control.Feedback type="invalid">
-            <ul>
-              {state.errors.writer_name?.map((msg) => (
-                <li key={msg}>{ERRORS[msg]}</li>
-              ))}
-            </ul>
-          </Form.Control.Feedback>
         </Col>
       </Form.Group>
-
-      {auth.isEditor() && (
-        <Form.Group as={Row} controlId="profileIsEditor">
+      {auth.user?.is_editor && (
+        <Form.Group as={Row} controlId="isEditor">
           <Form.Label column sm={2}>
-            Role
+            Status:
```

Comment:
I think Role is better than Status here --- it sounds more natural, and is consistent with Wordpress conventions + the backend.

Reply:
https://github.com/UWmathNEWS/slugline-web/pull/13/commits/53ece12a397300e594d32c8dc8ce2d7c9859a9ed